packages/devextreme/js/__internal/ui/form/m_form.layout_manager.utils.ts

```diff
@@ -153,10 +196,46 @@ function _convertToEditorOptions({
   const stylingMode = externalEditorOptions?.stylingMode || editorStylingMode;
   const useSpecificLabelOptions = EDITORS_WITH_SPECIFIC_LABELS.includes(editorType);
 
+  let editorOptionsWithDropDown: Partial<dxDropDownEditorOptions<dxTextBox>> = {};
```

Comment:
used component the editor is based on, but still options types are unknown - what component is best to use here?

Reply:
option method returns unknown,